README.md

```diff
@@ -16,28 +16,28 @@ workshop.
 
 ### Background slides:
 
-The accompanying slides for each module are linked below and briefly introduce
-the concepts underlying each module.
+The accompanying slides for each module are linked below and briefly introduce the concepts underlying each module.
 
-- [intro-to-R-tidyverse](https://drive.google.com/a/ccdatalab.org/file/d/11GXEddKwUBan1Z-NF_UM2HecckpODLKM/view?usp=sharing)
-- [RNA-seq](https://drive.google.com/a/ccdatalab.org/file/d/1A9gNDIuD_c3ppF2k6vY3b0VgSKZjchzp/view?usp=sharing)
-- [scRNA-seq](https://drive.google.com/a/ccdatalab.org/file/d/186niFprBKICNsF53WpIhKbiIMLawu-ms/view?usp=sharing)
-- [machine-learning](https://drive.google.com/a/ccdatalab.org/file/d/1tmX8sFDmnPpkRdkWQm-v3vtdGr6YWy-j/view?usp=sharing)
+- [Intro to R tidyverse](https://docs.google.com/presentation/d/1VwKLleg5RG4Ck_SWRgQhLOWmoguB9k5uhhmz1cdtea0/edit?usp=sharing)
```

Comment:
This slide is not quite as advertised, so I am renaming it... I wonder if we also want to rename the slide deck to remove the date, but that should not change the link


```suggestion
- [Intro to RStudio](https://docs.google.com/presentation/d/1VwKLleg5RG4Ck_SWRgQhLOWmoguB9k5uhhmz1cdtea0/edit?usp=sharing)
```